src/helpers/figures/charts/runtime/chart_data_extractor.ts

```diff
@@ -177,21 +213,45 @@ export function getLineChartData(
   return {
     dataSetsValues,
     axisFormats,
-    labels,
+    labels: applyLineBreakForLabels(labels),
     locale: getters.getLocale(),
     trendDataSetsValues,
     axisType,
   };
 }
 
+function applyLineBreakForLabels(labels: string[]): (string | string[])[] {
+  return labels.map((label) => (label.includes("\n") ? label.split("\n") : label));
```

Comment:
wait so you add newLines, and then you split  on the newLines to create an array ?  Why not create an array in the first place ? Or maybe I'm missing something